GoogleForCreators/web-stories-wp
# Story Locking: Ensure author users and below correctly see information

## Context



## Summary

Instead of using the embedded user rest api request, just hardcode the user data in the api response. 

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Create a story as a user with Author role. 
2. Save draft. 
3. Open another browser. 
4. Login as another user. 
5. Open draft story. 
6. See lock message.
7. Take over editing
8. As the Author user, see the other user's name & avatar in the takeover dialog


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12441
Fixes #12604
